featured-projects/README.md

```diff
@@ -0,0 +1,17 @@
+# Featured projects
+
+## Lotus
+Lotus is an implementation of the Filecoin Distributed Storage Network specification.
+You can find the Lotus Testground plans [here](https://github.com/filecoin-project/lotus/).
```

Comment:
The workflows there have been disabled for a couple of months so I think we can skip that.

Reply:
Unless they do not work anymore and we have no idea if they do since we're not running them. In this case, my thinking is that an untested example is worse than no example.

Reply:
That makes sense. I have removed lotus from the page for now.